lisa/sut_orchestrator/azure/common.py

```diff
@@ -400,6 +400,30 @@ def get_data_disk_size(
         raise LisaException("No data disk feature present.")
 
 
+@dataclass_json()
+@dataclass
+class DataDiskSchema:
+    caching_type: str = field(
+        default=constants.DATADISK_CACHING_TYPE_NONE,
+        metadata=schema.metadata(
+            validate=validate.OneOf(
+                [
```

Comment:
Maybe we can move these constants to `disk.py` and simplify the construct by doing something similar to `DiskType.get_disk_types` method.

Reply:
won't fix